python/ingestion/gcs_to_bq_util.py

```diff
@@ -221,15 +221,49 @@ def load_json_as_dataframe(gcs_bucket, filename, dtype=None):
     return frame
 
 
-def load_csv_as_dataframe_from_web(url, dtype=None, params=None, encoding=None):
+def load_csv_as_df_from_web(url, dtype=None, params=None, encoding=None):
     """Loads csv data from the provided url to a DataFrame.
        Expects the data to be in csv format, with the first row as the column
        names.
 
        url: url to download the csv file from"""
 
     url = requests.Request('GET', url, params=params).prepare().url
-    return pandas.read_csv(url, dtype=dtype, encoding=encoding)
+    return pd.read_csv(url, dtype=dtype, encoding=encoding)
+
+
+def load_csv_as_df_from_data_dir(directory, filename, dtype=None):
+    """Loads csv data from /data/{directory}/{filename} into a DataFrame.
+       Expects the data to be in csv format, with the first row as the column
+       names.
+
+    directory: directory within data to load from
+    filename: file to load the csv file from"""
+
+    file_path = os.path.join("data", directory, filename)
+
+    return pd.read_csv(file_path, dtype=dtype)
+
+
+def load_json_as_df_from_data_dir(directory, filename, dtype=None):
+    """Loads json data from /data/{directory}/{filename} into a DataFrame.
+       Expects the data to be in json format, with the first row as the column
+       names.
+
+    directory: directory within data to load from
+    filename: file to load the json file from"""
+
+    #  go up a level
+    path_parent = os.path.dirname(os.getcwd())
+    os.chdir(path_parent)
+
+    # go into /data
+    os.chdir("data")
+
+    # combine sub-dir and filename
+    file_path = os.path.join(directory, filename)
```

Comment:
Why change directories, why not just generate the entire filepath?

Reply:
I couldn't get that way working.... and this did...  and I was tired of debugging it by running the entire process lol. If you have a quick fix I'm happy to use it

Reply:
I'll give it a shot